corehq/apps/hqwebapp/templates/hqwebapp/base.html

```diff
@@ -510,6 +512,11 @@
           <script src="{% static 'hqwebapp/js/validators.ko.js' %}"></script>
         {% endif %}
 
+        {% if request.user.is_authenticated and request.user.is_active and is_saas_environment and show_mobile_ux_warning %}
```

Comment:
Since this is in base.html now, it needs to be compatible with requirejs:
- This if should also have a `and not requirejs_main` condition
- mobile_experience_warning.js needs to [declare its dependencies](https://github.com/dimagi/js-guide/blob/master/migrating.md#add-dependencies). This should be easy, I'd just test that the jquery cookie code works properly because I don't think that's been used on a requirejs page yet.
- Add `hqwebapp/js/mobile_experience_warning` to the set of modules included [here](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqwebapp/templates/hqwebapp/base.html#L404-L416)
It'd be good to test this on a page that uses requirejs, preferably on staging - the dashboard is an easy one.